### DISCUSSION TITLE
tmux system clipboard yank errors

### DISCUSSION THREAD

**Discussion by nightshade427** [helix-editor/helix#7751] [Troubleshooting]
### Summary

I can paste from tmux system clipboard using ```clipboard-paste-before```. But copying with ```clipboard-yank``` errors. Looking at the code it looks like helix is using ```tmux load-buffer -w -```. My version of tmux doesnt seem to have a ```-w``` option for saving into the clipboard buffer. Wonder if removing that option will enable tmux copy to work in more tmux versions?

### Platform

Debian 11, tmux (tmux 3.1c)

### Terminal Emulator

termux

### Helix Version

helix 23.05 (7f5940be)

**Comment by archseer** [comment]
`-w` is pretty critical for OSC 52 support. tmux 3.2 was released over two years ago so I'd rather see users update.

**Comment by archseer** [comment]
We should make the clipboard commands configurable though, which would make it possible to work around this in config rather than needing to recompile.

**Comment by archseer** [comment]
https://github.com/helix-editor/helix/pull/1827 contained this but was abandoned https://github.com/helix-editor/helix/pull/1827
